packages/api/cms-api/src/file-uploads/file-uploads.resolver.ts

```diff
@@ -10,7 +12,17 @@ import { FileUploadsService } from "./file-uploads.service";
 @Resolver(() => FileUpload)
 @RequiredPermission("fileUploads", { skipScopeCheck: true })
 export class FileUploadsResolver {
-    constructor(private readonly fileUploadsService: FileUploadsService, @Inject(FILE_UPLOADS_CONFIG) private readonly config: FileUploadsConfig) {}
+    constructor(
+        private readonly fileUploadsService: FileUploadsService,
+        @Inject(FILE_UPLOADS_CONFIG) private readonly config: FileUploadsConfig,
+        @InjectRepository(FileUpload)
+        private readonly repository: EntityRepository<FileUpload>,
+    ) {}
+
+    @Query(() => FileUpload)
+    fileUpload(@Args("id", { type: () => ID }) id: string): Promise<FileUpload> {
+        return this.repository.findOneOrFail(id);
+    }
```

Comment:
I'm not sure if this is a good idea. This might cause data leaks since uploads might be related to a user.